cmd/rookctl/object/bucket.go

```diff
@@ -88,23 +93,19 @@ func listBuckets(c client.RookRestClient) (string, error) {
 }
 
 var bucketGetCmd = &cobra.Command{
-	Use:   "get [BucketName]",
-	Short: "Gets the details of a bucket in the cluster",
+	Use:   "get [ObjectStore] [BucketName]",
+	Short: "Gets the details of a bucket in the object store",
 }
 
 func getBucketEntry(cmd *cobra.Command, args []string) error {
 	rook.SetupLogging()
 
-	if len(args) == 0 {
-		return fmt.Errorf("Missing required argument BucketName")
-	}
-
-	if len(args) > 1 {
-		return fmt.Errorf("Too many arguments")
+	if err := checkObjectArgs(args, []string{"[BucketName]"}); err != nil {
+		return err
 	}
 
 	c := rook.NewRookNetworkRestClient()
-	out, err := getBucket(c, args[0])
+	out, err := getBucket(c, args[0], args[1])
```

Comment:
nit: for readability, i would assign args to variables so that it is clear what these args means